syntax = "proto3";
package DB.FoundationDB.Proto;

message MetadataDatabase {
    string name = 1;
    string sql = 2;
}
